storage/pkg/command/storagehome.go

```diff
@@ -142,6 +142,12 @@ func storageHomeConfigFromStruct(c *cli.Context, cfg *config.Config) map[string]
 			},
 		},
 	}
+	if cfg.Reva.StorageHome.ReadOnly {
+		gcfg := rcfg["grpc"].(map[string]interface{})
+		gcfg["interceptors"] = map[string]interface{}{
```

Comment:
This would replace an existing map. You will have to check if it exists and create a new map or append to the existing one. Move that to a function with a name that explains what it does, eg. 'appendInterceptor()' then we can use it to add arbitrary interceptors... Or maybe even generic `map[string]interface{}` things 🤔

Reply:
Yes. But the whole struct is created in L113. I don't see a case where there is already another interceptor.

I was digging into it but let us merge now. We will see what is needed when we have a real use case for more interceptors.